test/index.js

```diff
@@ -16,6 +16,8 @@ const reader = new MongoOplogReader({
   redisClient: redis.createClient()
 });
 
+reader.filter(oplogDocument => oplogDocument.op !== 'u'); // ignore 'update' operations
```

Comment:
If `oplogDocument` has no `op` attribute (**undefined**) this will return true.  Is this the desired behavior too?

Reply:
yes